ulog/log.go

```diff
@@ -186,6 +193,16 @@ func (l *baseLogger) fieldsConversion(keyVals ...interface{}) []zap.Field {
 				fields = append(fields, zap.Marshaler(key, value))
 			case fmt.Stringer:
 				fields = append(fields, zap.Stringer(key, value))
+			case stackTracer:
+				stack := value.StackTrace()
+				lines := make([]zap.Field, 0, len(stack))
+				for _, frame := range stack {
+					s := fmt.Sprintf("%s", frame)
+					l := fmt.Sprintf("line:%d", frame)
```

Comment:
That is orthogonal. We can still have them in separate fields:
case error:
line = fmt.Sprintf("%v", err)
fields = append(fields, zap.Nest("stacktrace", line), zap.Error(value))